src/uu/test/src/parser.rs

```diff
@@ -90,6 +90,24 @@ impl Symbol {
     }
 }
 
+impl std::fmt::Display for Symbol {
```

Comment:
could you please document this ?

Reply:
I think the comment could be a bit more descriptive, you could mention that it's for printing errors and that we try to match the format in which the symbol appears in the input.